src/Vocup/MainForm.cs

```diff
@@ -125,9 +125,45 @@ public void UnloadBook(bool fullUnload)
             }
         }
 
+        private void StoreSettings()
+        {
+            //
+            // save the forms location and size
+            //
```

Comment:
Your comment would be even better as an embedded documentation like this:

```suggestion
        /// <summary>
        ///  Save the location and size of this window.
        /// </summary>
        private void StoreSettings()
```

The same applies for `RestoreSettings()`.